opennms-dao/src/main/java/org/opennms/netmgt/dao/support/EventConfServiceHelper.java

```diff
@@ -70,6 +69,7 @@ public static Long saveEvent(EventConfEventDao eventConfEventDao, EventConfSourc
         eventConfEvent.setEventLabel(event.getEventLabel());
         eventConfEvent.setDescription(event.getDescr());
         eventConfEvent.setEnabled(true);
+        eventConfEvent.setSeverity(event.getSeverity());
```

Comment:
Can we set severity to be `Indeterminate` if at all incoming event severity is not set.